src/App.js

```diff
@@ -1,23 +1,25 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+// import logo from "./logo.sxvg";
```

Comment:
For 99% of your projects it is safe to remove the logo and code that is premade on your app.js, We usually just keep the div with the className of App